src/quick_start/browser_extension.md

```diff
@@ -0,0 +1,104 @@
+# TLSNotary Browser Extension <a name="browser"></a>
+
+In this Quick Start we will prove ownership of a Twitter account with TLSNotary's browser extension.
+
+## Run Notary Server
+
+If the notary server isn't running already, run the notary server (in this repository: <https://github.com/tlsnotary/tlsn.git>):
+
+```shell
+cd notary-server
+cargo run --release
+```
+
+The notary server will now be running in the background waiting for connections.
+
+## Install Browser Extension (Chrome/Brave)
+
+1. Download the browser extension from
+<https://github.com/tlsnotary/tlsn-extension/releases/download/0.0.1/tlsn-extension-0.0.1.zip>
```

Comment:
Should we not link to a specific version, but instead to the release page and ask people to use the latest version since we are asking to run the latest notary above

Reply:
Agree. I run into this issue, because the current HEAD of notary-server is not compatible with the `tlsn-extension-0.0.1`

We should also streamline the build process for the extension.

Reply:
@tsukino Is this possible?